example-streamer-uses/README.md

```diff
@@ -1,45 +1,80 @@
+# How to run "up-linux-streamer"
+
+### Run following command from root directory of project:
+```bash
+
```

Comment:
```suggestion
```

Reply:
I am assuming this suggested change is to remove extra newline from code block. It is done.